### ISSUE TITLE
foundry.js:2013 TypeError: Cannot read properties of null (reading 'visible')

### ISSUE THREAD

**Issue by ravingdragoon** [death-save/combat-carousel#115]
**Describe the bug**
The bug is stopping the Combat Carousel from appearing on my game. I have tried disabling all modules except Combat Carousel, libwrapper, socketlib, and Find the Culprit, but the error is still occurring.

**To Reproduce**
1. Go to 'an active scene with an active combat'
2. Click on 'N/A'
3. Scroll down to 'N/A'
4. See error in console and no combat carousel on the scene

**Expected behavior**
See the combat carousel

**Screenshots**
```
TypeError: Cannot read properties of null (reading 'visible')
[Detected 2 packages: combat-carousel, lib-wrapper]
    at combat-carousel.mjs:291
    at Array.filter (<anonymous>)
    at CombatCarousel.getData (combat-carousel.mjs:291)
    at CombatCarousel._render (foundry.js:2049)
    at Z.🎁call_wrapped [as call_wrapped] (libWrapper-wrapper.js:502)
    at CombatCarousel.🎁Application.prototype._render#lib-wrapper (listeners.js:110)
    at CombatCarousel.🎁Application.prototype._render#0 (libWrapper-wrapper.js:182)
    at CombatCarousel._render (combat-carousel.mjs:48)
    at CombatCarousel.render (foundry.js:2011)
    at Function._onReady (combat-carousel.mjs:412)
    at hooks.mjs:36
    at Function._call (eval at <anonymous> (listeners.js:91), <anonymous>:4:14)
    at Function.callAll (foundry.js:153)
    at Game.setupGame (foundry.js:4403)
    at async Game._initializeGameView (foundry.js:5213)
    at async Game.initialize (foundry.js:4286)
	(anonymous)	@	foundry.js:2013
Promise.catch (async)		
render	@	foundry.js:2011
_onReady	@	combat-carousel.mjs:412
(anonymous)	@	hooks.mjs:36
_call	@	VM277652:4
callAll	@	foundry.js:153
setupGame	@	foundry.js:4403
```

**Environment Info (please complete the following information):**
 - Foundry Version: 0.8.9
 - Game System / Version: dnd5e 1.5.6
 - CC Version 0.2.3
 - Browser Chrome
 - OS Windows 10
 - 
**Additional context**
This is an ongoing combat that is on round 6. The combat still appears in the combat tracker tab, but the carousel does not appear, and the toggle button does nothing (not even change the little open/close arrow in the corner)

**Comment by ravingdragoon** [comment]
Update, this is also occurring on Foundry V9 Stable.

**Comment by eclarke12** [comment]
can you please DM me a zip of the world in question? If you use discord you can send it to errational#2007

**Comment by ravingdragoon** [comment]
[ForgeVTT-export-curse-of-strahd.zip](https://github.com/death-save/combat-carousel/files/7891045/ForgeVTT-export-curse-of-strahd.zip)

**Comment by eclarke12** [comment]
Fixed by #146
